packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -249,6 +249,43 @@ const Conversation = WebexPlugin.extend({
     return activity;
   },
 
+
+  /**
+  * Gets an array of activities with an array of activity URLS
+  * @param {Array} activties
+  * @param {String} cluster
+  * @returns {Promise<Object>} Resolves with the activities
+  * TODO: add cluster functionality when clusters are ready
+  */
+  bulk_activities_fetch(activties, cluster) {
+    if (!cluster) {
+
+      const params = {
+        method: 'POST',
+        api: 'conversation',
+        resource: 'bulk_activities_fetch',
+        body: {
+          activityUrls
+        }
+      };
+
+      return this.webex.request(params)
+        .then((res) => {
+          const activitiesArr = [];
+
+          res.body.multistatus.forEach((statusData) => {
+            if (statusData.status === '200') {
+              activitiesArr.push(statusData.data.activity);
```

Comment:
same here

Reply:
You want to check if the fields exists?

Reply:
yes, so something like `if (statusData.status === '200' && statusData.data && statusData.data.activity)`

Reply:
okay I see